Assets/Scripts/Cards/Card.cs

```diff
@@ -3,23 +3,23 @@
 using UnityEngine;
 using UnityEngine.Events;
 
-public class Card : ScriptableObject {
+public abstract class Card : ScriptableObject {
 
 	public Sprite cardArt;
 
 	public string cardName;
 	public string type;
-	public string description;
+	public string abilities;
```

Comment:
I wonder if this should be a `List<string>` instead so we can have each ability stored in a separate string? We should talk to Karen and Dexter about how they'd prefer to keep abilities (or even if cards will have more than one?)